cou/commands.py

```diff
@@ -159,6 +159,13 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         action=argparse.BooleanOptionalAction,
         default=argparse.SUPPRESS,
     )
+    subcommand_common_opts_parser.add_argument(
+        "--set-noout",
+        help="Set ceph noout across the cluster before cloud upgrade.\n"
+        "Default to disable set noout.",
+        action=argparse.BooleanOptionalAction,
+        default=argparse.SUPPRESS,
+    )
```

Comment:
From my perspective this flag should be something like `--ignore-noout` and set as default to false. If the user pass this flag then we don't check the `noout` as pre-upgrade step and we don't check in the end if was disabled

Reply:
That's not the purpose of the post upgrade sanity check; the post upgrade sanity (e.g. check if a config option is reverted or not, check if the version of workload is okay or not, like a `cou lint` ) should always be run at the end of an upgrade  with or without `--set-noout`.

Reply:
I think this PR would be easier if there was two options independently of the naming of the flag:

- completely ignore noout (as it happens today)
- check as upstream recomendation

With the implementation of `post upgrade sanity check` I think the complexity increased for just running one sanity check (of course can increase the number in the future).

Solving as it seems that the recommended way won't run as default